Source/EventHorizon/Producer/EventHorizon.cs

```diff
@@ -149,9 +149,14 @@ async Task ProcessEventsThroughEventHorizon()
                         _cancellationTokenSource.Token).ConfigureAwait(false);
                     if (!tryGetStreamEvent.Success)
                     {
+                        var nextPosition = await publicEvents.GetNextStreamPosition(_cancellationTokenSource.Token).ConfigureAwait(false);
+                        if (!nextPosition.Success)
+                        {
+                            throw nextPosition.Exception;
```

Comment:
We could just wait a little bit and continue the loop here - but maybe this exception is caught and retried elsewhere?

Reply:
It is. This should be okay to do